File: src/java1/q3.java
//Write a program to find the number of occurrences of a character in a string without using loop?

package java1;
public class q3 {
    public static void main(String[] args) {
        String st="tanvi gupta";
        String ch="a";
        System.out.println(st.length() - st.replace(ch,"").length());

    }


}

File: Exercise1/q3.kt
package AssesmentOne

//Write a program to find the number of occurrences of a character in a string without using loop.
fun main(args: Array<String>) {
    var str = "hippopotamus"
    val character = "p"

    val num = (str.length - (str.replace(character,"")).length)
    println(num)

}